src/main/scala/dotty/dokka/translators/ScalaPageCreator.scala

```diff
@@ -393,7 +393,7 @@ class ScalaPageCreator(
         
 
         def contentForTypesInfo(c: DClass) =
-            val supertypes = c.parents
+            val supertypes = c.parents.values.flatMap(x => x)
```

Comment:
there is a `flatten` method that does exacrtly the same as `flatMap(x => x)`

Reply:
Will fix